add multilang support and update view

WinterSilence opened the pull request.

Commit:
add multilang support

birkir commented on classes/Kohana/Date.php:
```diff
@@ -514,12 +514,12 @@ public static function fuzzy_span($timestamp, $local_timestamp = NULL)
 		{
 			// This is in the past
-			return $span.' ago';
+			return __($span).' '.__('ago');
```
Shouldn't this be done with parameters?
__(':span ago', array(':span' => __($span)));
as some languages will have "ago", before the actual $span.

WinterSilence replied:
what would you suggest?

birkir replied:
http://kohanaframework.org/3.3/guide/kohana/files/i18n#i18n-variables
I wrote an example above.

WinterSilence replied:
ups :) not immediately understand you. make changes now. thanks!

Commit:
fix multilang

Commit:
View extension and small doc  fixes

Replace View::set_filename on View::filename (gets\sets)
Add View::clear for delete variables after render for fast rendering of
similar patterns without the call View::factory every time

Commit:
View fixes

Commit:
Fix and extends controller guide
